conda/notices/core.py

```diff
@@ -44,7 +44,11 @@ def retrieve_notices(
 
     # We always want to modify the mtime attribute of the file if we are trying to retrieve notices
     # This is used later in "is_channel_notices_cache_expired"
-    cache_file.touch()
+    try:
```

Comment:
This catches the specific error I had, but I think there other i/o operations below that would also fail if the cache file cannot be changed/read/etc., right?

Reply:
There's a get first writable function used in subdir_data that might be the thing.

Reply:
@beckermr, good catch. I still want to figure out how to properly test this so I rest assured I know what I'm doing 😅. .